### PR TITLE
Updates in few pages

### PR DESCRIPTION
**Pull request by Nikhil-Ladha**
Signed-off-by: Nikhil-Ladha <nikhilladha1999@gmail.com>

**Description**

This PR updates blogs, books, news, related posts, landscape, and a few other sections.
Also, clears a few unwanted CSS.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by Jashpatel1** [comment]
The top margins for blog grid and blog list are different. Please have a look into it.

File: src/components/Card/Card.style.js [hunk 1]
```diff
@@ -2,9 +2,11 @@ import styled from "styled-components";
 
 export const CardWrapper = styled.div`
     .post-block{
-        margin-bottom: 30px;
-        border-radius: 3px;
+        margin-bottom: 2rem;
```
Comment (leecalcote) [inline src/components/Card/Card.style.js:5]: YES

File: src/components/Card/Card.style.js [hunk 2]
```diff
@@ -2,9 +2,11 @@ import styled from "styled-components";
...
+        box-shadow: 0px 1px 10px -6px ${props => props.theme.shadowLightColor};
```
Comment (leecalcote) [inline src/components/Card/Card.style.js:6]: YES

**Comment by Nikhil-Ladha** [comment]
> The top margins for blog grid and blog list are different. Please have a look into it.

Done.

**Review by Jashpatel1 — COMMENTED** [review]
In the news card, the external link icon can be moved away from see more.

**Comment by chandrashritii** [comment]
**Suggestions**

- Go with american date formats (MM-DD-YYYY)
- Expand the margins for all the box images to fit the box width.
- [Books Page] Box Shadow isn’t being applied to the outermost element; isn’t visible. 
- Remove unnecessary padding
- Table on the Landscape page - Add suitable padding
- Increase font size for ‘Service Mesh Timeline’ header

**Comment by Nikhil-Ladha** [comment]
> **Suggestions**
> 
> * Go with american date formats (MM-DD-YYYY)
> * Expand the margins for all the box images to fit the box width.
> * [Books Page] Box Shadow isn’t being applied to the outermost element; isn’t visible.
> * Remove unnecessary padding
> * Table on the Landscape page - Add suitable padding
> * Increase font size for ‘Service Mesh Timeline’ header

Thank you so much @chandrashritii for adding the comments here itself from the meeting minutes 🙂👌🏼

**Comment by Nikhil-Ladha** [comment]
We want the date in which of the following format?

- MM-DD-YYYY
Screenshot from 2020-12-22 14-05-32

- MMM Do, YYYY
Screenshot from 2020-12-22 14-05-45

@leecalcote

**Comment by leecalcote** [comment]
Thanks for asking. The second one: MMM Do, YYYY

**Comment by Nikhil-Ladha** [comment]
Sorry for making the PR so big 😆 . That was a hell of a cleanup!!
I think it can be safely merged and it will not break anything. If in case any issue occurs, a separate PR can be made.

**Review by Jashpatel1 — APPROVED** [review]

**Pull request merged**